google-ads-examples/build.gradle

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This is the build file for the google-ads-examples subproject. It depends on
+ * the google-ads subproject.
+ *
+ * This project contains examples for using the Google Ads API. You can launch
+ * examples by navigating to the google-ads-examples folder and running the
+ * provided runExample gradle task. Note that we recommend running with the
+ * quiet flag (-q) to reduce screen clutter:
+ *     -- Mac/Linux --
+ *     ./gradlew -q runExample --example='basicoperations.GetCampaigns --customerId 1234567890'
+ *
+ *     -- Windows --
+ *     gradlew -q runExample --example='basicoperations.GetCampaigns --customerId 1234567890'
+ */
+
+plugins {
+    id 'com.google.api-ads.java-conventions'
+}
+
+dependencies {
+    implementation project(':google-ads')
+    implementation 'org.apache.commons:commons-lang3:3.11'
+    implementation 'com.beust:jcommander:1.72'
+    implementation 'joda-time:joda-time:2.8.2'
+    runtimeOnly 'org.apache.logging.log4j:log4j-slf4j-impl:2.11.1'
+    testImplementation 'org.mockito:mockito-core:2.27.0'
+}
+
+description = 'Google Ads API client library for Java examples'
+
+public class ExampleRunnerTask extends JavaExec {
```

Comment:
Is there a way to have this for the migration examples project as well, without duplication?

Reply:
Good point, I didn't explicitly test those. I'll adjust and test.

Reply:
The migration examples should of course still be runnable from IntelliJ.

Reply:
Will test moving the task code into the top-level build files, then filtering for which subproject to apply them to.

Reply:
Restructured; task code is now in `buildSrc`, and there are now two executable tasks: `runExample`, as before, and `runMigrationExample`.